packages/paste-website/src/pages/roadmap/index.mdx

```diff
@@ -33,27 +33,86 @@ export const pageQuery = graphql`
 
 ## Release 0.4
 
-### Release Date: December 6, 2019
+Release date: December 3, 2019
 
-- Flex Tokens will be added to our Design System Sketch files
-- Theme Switcher will work as intended, allowing all styles to be viewable based on SendGrid and Console Themes.
+**Components**
+
+- Flex tokens will be added to our Design System Sketch files
 - Buttons will be productionized 🎉 and usable on SendGrid / Console.
 - Anchors will be productionized too! 🔗
-- Icon Systems will also be on the production train 🚝
-- Alerts will be design ready, and built in an upcoming release.
-- Grids will be design ready, and built in an upcoming release.
-- Form Inputs (Inputs and Textareas) will be design ready, and built in an upcoming release.
+- The icon system will also be on the production train 🚝
+
+**Documentation**
+
+- Theme Switcher will work as intended, allowing all styles to be viewable based on SendGrid and Console Themes.
+
+**Behind the scenes**
+
+- Alerts will be design-ready and built in an upcoming release. 🔔
+- Grids will be design-ready and built in an upcoming release. 
+
+---
 
 ## Release 0.5
 
-### Release Date: January 17, 2020
+Release date: January 14, 2020
+
+**Components**
+
+- Ship an Alert component. 🔔
+- Ship a Grid component.
+- Ship more Typography components (Heading and Lists).
+
+**Documentation**
+
+- Add an icon list to this website with more design guidelines.
+
+**Behind the scenes**
+
+- Design and ship basic form field components (Label, Input, Textarea, and Inline error). 📝
+- Design a Toast component. 🍞
+- Design more form field components (Select, Checkbox and Checkbox group, Radio and Radio Group, and Toggle). 📝
+- Design a Button group component.
+- Apply unified design language for alpha.
+- Implement continuous deployment of Paste packages. 
+
+---
```

Comment:
After restructuring roadmap releases to Design / Eng / Other, add TBD to sections where there is no content in each subheading.